js/exporter/jspdf/v3/rows_generator.js

```diff
@@ -43,7 +43,7 @@ function generateRowsInfo(dataProvider, dataGrid, headerStyles) {
                 columns,
                 rowType,
                 colCount: columns.length,
-                backgroundColor: (rowType === 'header') ? headerStyles?.backgroundColor : undefined
+                backgroundColor: (rowType === 'header') ? rowOptions?.headerStyles?.backgroundColor : undefined
```

Comment:
A more complex object will be passed:

`headerStyles?.backgroundColor -> rowOptions?.headerStyles?.backgroundColor 
`

Can we pass the used 'headerBackgroundColor' value only?

Reply:
Done